### ISSUE TITLE
NextGen: Chosen Page Inject template not selected in properties dialog

### ISSUE THREAD

**Issue by paulhibbitts** [getgrav/grav-premium-issues#24]
I've noticed that for the Page Inject shortcodes in NextGen, the chosen template is displayed inline but not by default in the properties dialog:

<img width="1440" alt="2021-01-25_10-01-11" src="https://user-images.githubusercontent.com/1812771/105746734-aba9df80-5ef4-11eb-92c7-44db95790938.png">

<img width="1440" alt="2021-01-25_10-01-22" src="https://user-images.githubusercontent.com/1812771/105746749-b06e9380-5ef4-11eb-8346-01c13ec05907.png">

In case I am using an incorrect syntax, here is my markdown:
```
[fa="book"][/fa][plugin:page-inject](/211/weekly-readings/week-02-1?template=partials/embedlycardlinkonly)  
```

Please let me know if any more info is helpful etc.

Thanks very much,
Paul

**Comment by w00fz** [comment]
Are you suggesting that `Default` should always be the selected template and it should never display as "**No template selected**"?

**Comment by paulhibbitts** [comment]
No, I am wondering if a) are templates in `Partial` supported and then b) if the current partial template can then be initially selected in the properties dialog (like non-partials are)? Thanks!

**Comment by w00fz** [comment]
Got it, I think what you are seeing is an issue with the parsing again, caused by the fact that the template has a forward slash in it `/` which most likely is getting interpreted as deeper path.

Need to investigate this case.

**Comment by paulhibbitts** [comment]
Thanks @w00fz , please let me know if I can be of any help when convenient.

**Comment by w00fz** [comment]
Alright @paulhibbitts so this is what's going on with this. It was a mistake for us to have the list of pages as a dropdown. Right now Grav is able to collect page types and modular types but really what Page Inject in combination with NextGen wants is to gather the full twig templates list available to pick from.

This is of course doable but it will require a Grav change and in fact even just that won't be enough. We would need to implement some sort of type-ahead logic for NextGen so that we don't populate a dropdown with an immense amount of templates to chose from but instead you get a filtered list as you type.

Ultimately this is where we would like to head to on a future release but at this time with NextGen and Page Inject being the sole place of this functionality being requested, I don't think it's worth for us to implement. What I did instead was to back away from the dropdown entirely and instead turn the dropdown into a text input, this way you can manually specify whichever template you know you want to use and it won't get lost any longer because not available in the list.

This is now available as an update in Page Inject 1.4.4.

Cheers!

**Comment by paulhibbitts** [comment]
Hi @w00fz , your immediate approach sounds very reasonable🙂 I've also just tested it and it works like a charm - thank you!
